src/app/components/top/solicitudes/detalleSolicitud.html

```diff
@@ -1,159 +1,139 @@
-<div class="mt-3">
-    <div class="row"
-         *ngIf="prestacionSeleccionada.estadoActual.tipo === 'rechazada'">
-        <div class="col">
-            <!-- Cuando la solicitud fue rechazada muestra los datos del rechazo -->
-            <b>Observaciones</b>
-            <br>
+<plex-detail size="md" direction="row">
+    <img [mpiFotoPaciente]="prestacionSeleccionada?.paciente" />
+    <plex-badge type="warning">
+        {{prestacionSeleccionada.solicitud.fecha
+            | fecha}}
+    </plex-badge>
+    <plex-badge type="danger"
+                *ngIf="prestacionSeleccionada.solicitud.registros[0].valor.solicitudPrestacion.prioridad === 'prioritario'">
+        {{prestacionSeleccionada.solicitud.registros[0].valor.solicitudPrestacion.prioridad}}
+    </plex-badge>
+    <plex-badge type="info"
+                *ngIf="!prestacionSeleccionada.solicitud.turno && prestacionSeleccionada.estadoActual.tipo !== 'rechazada'">
```

Comment:
[Sugerencia] este tipo de if se pueden hacer más legibles con: 
```suggestion
                *ngIf="noRechazada">
                // .ts
                get noRechazada() {
    return !this.prestacionSeleccionada.solicitud.turno && this.prestacionSeleccionada.estadoActual.tipo !== 'rechazada';
}
```